tests/test_template.py

```
@@ -373,12 +461,403 @@ def test_rust_template_custom_package(copier: CopierFixture, tmp_path: Path) ->
+    [
+        ("tooling-pure", "ToolingPure", "tooling_pure", False),
+        ("tooling-rust", "ToolingRust", "tooling_rust", True),
+    ],
+)
+def test_generated_tooling_contracts(
+    copier: CopierFixture,
+    tmp_path: Path,
+    target_dir: str,
+    project_name: str,
+    package_name: str,
+    use_rust: bool,
```

Ruff (0.15.14)

[warning] Boolean-typed positional argument in function definition

(FBT001)